### DISCUSSION TITLE
The "image-height" in my blog head matter crops the image instaed of scaling it. Is there a workaround?

### DISCUSSION THREAD

**Discussion by davidcannatella** [quarto-dev/quarto-cli#8038] [Q&A]
title: "Blog"
subtitle: "This is my blog"
listing:
  contents: posts
  type: default
  image-height: 30px  #  <-- crops the image, but no scaling
page-layout: full

**Comment by mcanouil** [comment]
Could you share a small self-contained "working" (reproducible) example to work with, _i.e._, a complete Quarto document or a Git repository? Thanks.

You can share a Quarto document using the following syntax, _i.e._, using more backticks than you have in your document (usually four ` ```` `).

`````md
````qmd
---
title: "Reproducible Quarto Document"
format: html
---

This is a reproducible Quarto document using `format: html`.
It is written in Markdown and contains embedded R code.
When you run the code, it will produce a plot.

```{r}
plot(cars)
```

![A placeholder image](https://placehold.co/600x400.png)

The end.
````
`````

Likely related to:
- #2999

**Reply by davidpassmore** [reply]
I, too, have this question.

**Reply by mcanouil** [reply]
@davidpassmore Read the above reply. Unfortunately, your comment does not bring any information for anyone to provide more help.
